lib/msal-core/src/error/ClaimsRequiredError.ts

```diff
@@ -21,13 +21,20 @@
   * OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
   */
 
+import { InteractionRequiredAuthError } from "./InteractionRequiredAuthError";
+
 /**
  * @hidden
+ *
+ * Error thrown when the client must provide additional proof to acquire a token. This will be used for conditional access cases.
  */
-export class ErrorMessage {
-  static get authorityUriInvalidPath(): string { return "AuthorityUriInvalidPath"; }
-  static get authorityUriInsecure(): string { return "AuthorityUriInsecure"; }
-  static get invalidAuthorityType(): string { return "InvalidAuthorityType"; }
-  static get unsupportedAuthorityValidation(): string { return "UnsupportedAuthorityValidation"; }
-  static get b2cAuthorityUriInvalidPath(): string { return "B2cAuthorityUriInvalidPath"; }
+export class ClaimsRequiredAuthError extends InteractionRequiredAuthError {
+    claimsRequired: string;
+    constructor(errorCode: string, errorMessage: string, claimsRequired: string) {
```

Comment:
do we not have any claims required error?

Reply:
Not currently, coming in a future feature when we implement Conditional Access.

Reply:
Is this a feature we are eventually going to implement? or that you have code ready to review for?
Its such a small amount of code that if we are not using it, I would prefer we just write it when we have a use case. 

If for some un forseen reason feature priorities change or something like that, pushing unused code becomes debt

Reply:
There is a lot of work that is put in understanding Claims @DarylThayil . We have a use case and it is implemented in most MSALs. We are currently in the middle of untangling some requirements and should be able to implement this soon.

Reply:
@sameerag  my feedback was just to avoid comitting code that we aren't using yet.  It's fairly simple to add the 20 lines to create the error class when we need it in code